.gitignore

```diff
@@ -15,3 +15,4 @@ z_local.sbt
 /.ensime_cache
 .tags
 .metaserver
+test-output
```

Comment:
`TestNG` creates a `test-output` directory under `reactive-streams/`.  I couldn't find a way of moving it under `target`.